src-java/kilda-model/src/main/java/org/openkilda/adapter/FlowSourceAdapter.java

```diff
@@ -37,4 +40,9 @@ public FlowEndpoint getEndpoint() {
     public boolean isMultiTableSegment() {
         return flow.isSrcWithMultiTable();
     }
+
+    @Override
+    public boolean isPrimaryEgressPath(@NonNull PathId pathId) {
+        return pathId.equals(flow.getForwardPathId());
```

Comment:
This path goes from source to dest i.e. it is forward path. So from the "source" flow side view, it is egress path, because it starts at the "source" side and goes somewhere else (in our case to the dest side of the flow).

Why it must be "ingress"?